test/unit/gateways/cyber_source_rest_test.rb

```diff
@@ -187,6 +203,34 @@ def test_add_shipping_address
     assert_equal '4158880000', address[:phoneNumber]
   end
 
+  def test_authorize_network_token_visa
```

Comment:
I know these tickets aren't together but could you please add tests for stored credentials just to ensure that we are properly sending all the fields.

Reply:
There's another PR to update the stored credential mappings!